src/pages/hacktoberfest23.tsx

```diff
@@ -0,0 +1,18 @@
+import Header from '@/components/Header';
+import Script from 'next/script';
+
+export default function About() {
```

Comment:
for this page maybe we should add a custom container because the form banner is cutoff on smaller devices.

Reply:
![image](https://github.com/GalsenDev221/website/assets/46305144/65dd8c97-c95f-4292-a109-b8fb11ed37ec)

On smaller screen the top image is not fully displayed.

Reply:
Yes I noticed the image cuts off when you resize it.
Maybe it's the iframe that makes it like that?
But I think the most important thing is that we are just the form that appears.

Reply:
Yes indeed